cmd/cluster/validatepullsecret.go

```diff
@@ -0,0 +1,100 @@
+package cluster
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var BackplaneClusterAdmin = "backplane-cluster-admin"
+
+func newCmdValidatePullSecret(kubeCli client.Client, flags *genericclioptions.ConfigFlags) *cobra.Command {
+	return &cobra.Command{
+		Use:               "validate-pull-secret",
+		Short:             "Checks if the pull secret email matches the owner email",
```

Comment:
Same docs callout

Reply:
per our slack conversation, I've updated the usage and added a Long property here.